src/Lean/Parser/Module.lean

```diff
@@ -69,7 +75,7 @@ def parseHeader (inputCtx : InputContext) : IO (Syntax × ModuleParserState × M
   let stx := if s.stxStack.isEmpty then .missing else s.stxStack.back
   match s.errorMsg with
   | some errorMsg =>
-    let msg := mkErrorMessage inputCtx s errorMsg
+    let msg := mkErrorMessage inputCtx s.pos s.stxStack errorMsg
     pure (stx, { pos := s.pos, recovering := true }, { : MessageLog }.add msg)
```

Comment:
Let's make sure we don't ignore any recovery errors here even if we don't have any right now.

Reply:
Should be good now!